### DISCUSSION TITLE
Wishes =)

### DISCUSSION THREAD

**Discussion by BahamutRU** [innightwolfsleep/llm_telegram_bot#55] [Ideas]
1. Image recognition. Perhaps this would be the most interesting thing a bot can do after recording voice messages.
2. ✅ Creating images using sd_api_pictures || Creating images using sd_api_pictures_tag_injection is a very useful function in itself, it's a pity that it requires a lot of context. However, even purely drawing bots are a great thing, as for me.
3. Voice message recognition.
4. Multiple instance? If I run two copy with two settings file with two extensions — I have an error. I can run in docker, or virtualization, but I'm lazy. =)

I do not know how difficult it is to add to the bot, I just made my personal (sorry=) top features that the bot lacks.
UPD 30.07.2023

**Comment by innightwolfsleep** [comment]
Creating images using sd_api_pictures✅

**Comment by ExperTnot** [comment]
Id love to be able to use voice messages! I use whisper ai with a telegram bot i have. i am not smart enough to add it to this extension though. I can share my code for my whisper implementation if anyone needs it and wants to try to add it to this one
